Menu/Menu.js

```diff
@@ -33,3 +33,52 @@ let menuItems = [
   Step 6: add the menu component to the DOM.
   
 */
+
+
+//component [...]
+const menuButton = document.querySelector('.menu-button');
+/*new update on header*/
+const header = document.querySelector('.header')
+
+//adding the menu component to the DOM
+menuItems.forEach(add => {
+  menuButton.appendChild(menuCreator(add.arr));
+})
+
+function menuCreator(arr) {
+  const menuContainer = document.createElement('div')
+  const menuC = document.createElement('div')
```

Comment:
Probably won't need line 50 since you have a container above and all the items are going into a ul